<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <arg name="move_group" default="manipulator"/>
  <arg name="ee_link" default="iiwa_link_ee"/>
  <arg name="ros_rate" default="0.1"/>
  <arg name="planner_id" default="RRTConnectkConfigDefault"/>
  <arg name="reference_frame" default="arm_mount_link"/>

  <!-- Here we call the ROS node we have written, with the parameters on top   -->
  <node name="CommandRobotMoveit" pkg="iiwa_examples" type="drawing_moveit" respawn="false" output="screen">
    <param name="move_group" value="$(arg move_group)"/>
    <param name="ee_link" value="$(arg ee_link)"/>
    <param name="ros_rate" value="$(arg ros_rate)"/>
    <param name="planner_id" value="$(arg planner_id)"/>
    <param name="reference_frame" value="$(arg reference_frame)"/>
  </node>

</launch>
